src/process/run.cr

```diff
@@ -110,12 +133,23 @@ class Process
     fork_error.try &.close
   end
 
-  # Waits for this process to complete.
+  def kill sig = Signal::TERM
```

Comment:
A simple doc comment would be nice, perhaps just "Send a signal to the process, see Process.kill"